.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -702,10 +726,14 @@ stages:
     - script: tracer\build.cmd BuildDdDotnet
       displayName: Build dd-dotnet
 
-    - publish: $(artifacts)/dd-dotnet/win-x64
+    - publish: $(artifacts)/dd-dotnet/win-x64/dd-dotnet.exe
```

Comment:
It's not really related, to this PR, but the yaml we _generally_ have dependencies listed higher than dependent stages, so it would be nice to move the `build_dd_*` stages _above_ the packaging stages. 

Mostly it makes it easier when selecting a subset of stages to run

Reply:
👍